### PR TITLE
Add Serbian translation

### PR DESCRIPTION
**Pull request by eevan78**
Here is the UI translation to Serbian.

### CODE REVIEW

**Review by deluan — COMMENTED** [review]
Thanks for the contribution. Check the comment below. Otherwise looks good.

File: resources/i18n/sr.json [hunk 1]
```diff
@@ -0,0 +1,466 @@
+
+{
+    "languageName": "српски",
```
[the remark this answers is not in the document]
Comment (deluan) [reply resources/i18n/sr.json:3]: Screenshot 2024-09-19 at 1 58 50 PM

[the remark this answers is not in the document]
Comment (eevan78) [reply resources/i18n/sr.json:3]: Hello, I'm a native speaker, Serbian is my mothertongue :)
This is how you name languages in Serbian. As an adjective before the noun language (језик).
You don't have to state jeзик when it's obvious that this is a list of language names.

Here is the example for the Google search settings:

Screenshot 2024-09-20 093354

The popup title is Display language. Look at the French. It should be француски језик, just the word језик is omitted, as it is clear that this is the list of language names...

I hope that I managed to explain properly.

[the remark this answers is not in the document]
Comment (deluan) [reply resources/i18n/sr.json:3]: Awesome! Thanks for replying.

**Pull request merged**